pkg/log/zap/zap_test.go

```diff
@@ -217,7 +217,8 @@ var _ = Describe("Zap logger setup", func() {
 				Expect(json.Unmarshal(outRaw, &res)).To(Succeed())
 
 				Expect(res).To(HaveKeyWithValue("thing", map[string]interface{}{
-					"name": name.Name,
+					"Name":      name.Name,
+					"Namespace": "",
```

Comment:
Why is this needed?

Reply:
Sorry, this can be ignored and I will fix it properly. Currently something in 0.27 breaks marshaling and these tests fail (see the CI results)